pkgs/build-support/release/default.nix

```diff
@@ -37,26 +42,6 @@ rec {
       // args
     );
 
-  makeSourceTarball = sourceTarball; # compatibility
-
-  binaryTarball =
-    args:
-    import ./binary-tarball.nix (
-      {
-        inherit lib stdenv;
-      }
-      // args
-    );
-
-  mvnBuild =
-    args:
-    import ./maven-build.nix (
-      {
-        inherit lib stdenv;
-      }
-      // args
-    );
```

Comment:
Are these broken, or is this just on a principle of “nothing is using them and it seems like nobody cares”? (I don’t mind dropping them, to be clear.)